cfn.yaml

```diff
@@ -69,16 +69,21 @@ Resources:
               Action:
                 - ssm:GetParametersByPath
               Resource: !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${Stage}/${Stack}/${App}
-        - PolicyName: uploadBuildOutputToS3
+        - PolicyName: s3BuildOutput
           PolicyDocument:
             Statement:
-              Effect: Allow
-              Action:
-                - s3:PutObject
-                - s3:PutObjectAcl
-              Resource: !Sub
-                - arn:aws:s3:::static-content-dist/${Path}*
-                - Path: !FindInMap [StageVariables, !Ref Stage, UploadPath]
+              - Effect: Allow
+                Action:
+                  - s3:PutObject
+                  - s3:PutObjectAcl
+                  - s3:GetObject
+                Resource: !Sub
+                  - arn:aws:s3:::static-content-dist/${Path}*
+                  - Path: !FindInMap [StageVariables, !Ref Stage, UploadPath]
+              - Effect: Allow
+                Action:
+                  - s3:ListBucket
```

Comment:
This allows us to determine when an object is missing ([otherwise we'll get a 403](https://aws.amazon.com/premiumsupport/knowledge-center/s3-troubleshoot-403/)).